app/src/main/java/cm/aptoide/pt/wallet/WalletInstallPresenter.kt

```diff
@@ -1,22 +1,50 @@
 package cm.aptoide.pt.wallet
 
 import android.os.Build
+import cm.aptoide.pt.actions.PermissionManager
+import cm.aptoide.pt.actions.PermissionService
 import cm.aptoide.pt.presenter.Presenter
 import cm.aptoide.pt.presenter.View
-import cm.aptoide.pt.promotions.PromotionsManager
 import cm.aptoide.pt.promotions.WalletApp
+import rx.Completable
 import rx.Observable
 import rx.Scheduler
+import rx.schedulers.Schedulers
 
 class WalletInstallPresenter(val view: WalletInstallView,
```

Comment:
Since we didn't discuss how codestyle should be implemented, I don't think it is worth it to be changing it everywhere, having the risk of having to change it back.